docs/ntd-cli.md

```
@@ -513,6 +513,62 @@ ntd loop execution get <EXECUTION_ID>
+```
+
+**选项：**
+
+| 选项 | 说明 |
+|------|------|
+| `<EXECUTION_ID>` | 必填，loop execution 主键 |
+| `--json` | 输出原始 JSON（默认是人类可读黑板视图） |
+
+**示例输出**：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)